app/assets/javascripts/discourse/app/instance-initializers/enable-emoji.gjs

```diff
@@ -14,8 +16,21 @@ export default {
         toolbar.addButton({
           id: "emoji",
           group: "extras",
-          icon: "far-face-smile",
-          action: () => toolbar.context.send("emoji"),
+          icon: "discourse-emojis",
+          sendAction: () => {
+            const menu = api.container.lookup("service:menu");
+            menu.show(document.querySelector(".insert-emoji"), {
```

Comment:
`.insert-emoji` seems like quite a general selector, which could clash with something outside the composer. Could we tighten the scope a little, so we're sure it's the toolbar button being targetted?

Reply:
done